src/ec/packages/ec-component-social-media-follow/__snapshots__/social-media-follow.test.js.snap

```diff
@@ -144,23 +150,177 @@ exports[`EC - Social Media Follow Horizontal renders correctly 1`] = `
+>
+  <p
+    class="ecl-social-media-follow__description"
+  >
+    Follow the latest progress and learn more about getting involved.
+  </p>
+  <ul
+    class="ecl-social-media-follow__list"
+  >
+    <li
+      class="ecl-social-media-follow__item"
+    >
+      <a
+        class="ecl-link ecl-link--standalone ecl-link--icon ecl-link--icon-before ecl-social-media-follow__link"
+        href="/example"
+      >
+        <svg
+          aria-hidden="true"
+          class="ecl-icon ecl-icon--xl ecl-social-media-follow__icon ecl-link__icon"
+          focusable="false"
+        >
+          <use
+            xlink:href="example#twitter"
+          />
+        </svg>
+        <svg
+          aria-hidden="true"
+          class="ecl-icon ecl-icon--xl ecl-social-media-follow__icon-hover ecl-social-media-follow__icon ecl-link__icon"
+          focusable="false"
+        >
+          <use
+            xlink:href="example#twitter_hover"
+          />
+        </svg>
+         
+        <span
+          class="ecl-link__label"
+        >
+          Twitter
+        </span>
+      </a>
+    </li>
+    <li
+      class="ecl-social-media-follow__item"
+    >
+      <a
+        class="ecl-link ecl-link--standalone ecl-link--icon ecl-link--icon-before ecl-social-media-follow__link"
+        href="/example"
+      >
+        <svg
+          aria-hidden="true"
+          class="ecl-icon ecl-icon--xl ecl-social-media-follow__icon ecl-link__icon"
+          focusable="false"
+        >
+          <use
+            xlink:href="example#facebook"
+          />
+        </svg>
+        <svg
+          aria-hidden="true"
+          class="ecl-icon ecl-icon--xl ecl-social-media-follow__icon-hover ecl-social-media-follow__icon ecl-link__icon"
+          focusable="false"
+        >
+          <use
+            xlink:href="example#facebook_hover"
+          />
+        </svg>
+         
+        <span
+          class="ecl-link__label"
+        >
+          Facebook
+        </span>
+      </a>
+    </li>
+    <li
+      class="ecl-social-media-follow__item"
+    >
+      <a
+        class="ecl-link ecl-link--standalone ecl-link--icon ecl-link--icon-before ecl-social-media-follow__link"
         href="/example"
       >
+        <svg
+          aria-hidden="true"
+          class="ecl-icon ecl-icon--xl ecl-social-media-follow__icon ecl-link__icon"
+          focusable="false"
+        >
+          <use
+            xlink:href="example#gplus"
+          />
+        </svg>
+        <svg
+          aria-hidden="true"
+          class="ecl-icon ecl-icon--xl ecl-social-media-follow__icon-hover ecl-social-media-follow__icon ecl-link__icon"
+          focusable="false"
+        >
+          <use
+            xlink:href="example#gplus_hover"
+          />
+        </svg>
+         
         <span
           class="ecl-link__label"
         >
-          Other social networks
+          Google+
```

Comment:
As we are there, we can already replaced Google+ network by Instagram (we just did it on ECL)

Reply:
oooook

Reply:
we will release ECL 2.2.0 this afternooon, so we can already do it here

Reply:
How? :)